docs/Getting-Started/Private-Network-Quickstart.md

```diff
@@ -296,21 +314,22 @@ Install Truffle :
 npm install -g truffle
 ```
 
-!!!note
-    `npm` requires `sudo` on Linux.  
+!!! note
```

Comment:
with or without space between `!!!` and `note` it works the same. We could fix all blocks to be consistent in one way or the other, my favourite would be no space.

Reply:
Agreed. Added this to our Styleguide.